test/specs/crossHomePage.js

```diff
@@ -0,0 +1,30 @@
+const SELECTORS={
+  logoLink: '.logo',
+  howToLink: '.how-to-use',
+  infoLink: '.info',
+}
+
+class crossHomePage{
+  open(){
+    browser.url('http://localhost:3000/');
```

Comment:
`browser.url()` can use the `baseUrl` that is set up in the `wdio.conf` file, no need to put this whole path.
Just use `browser.url('/')`